scripts/converter.js

```diff
@@ -0,0 +1,5 @@
+class Converter {
```

Comment:
This class name makes me wonder; what are we going to convert?

For simple use cases like these I tend to opt for a simple utility function, as a class does not offer any additional benefits.

```javascript
function stringToArray(string, separator) {
  return string.split(seperator);
}
```

Reply:
Maybe we'll use it for future conversions? I think I had somewhere else to use this class aswell for a new conversion but I forgot to do that... I don't mind having all the converters in the same place though?

Reply:
You could still have all the functions in the same file. :)

The reason you use a class (usually) is to utilise `this`.

Imagine if the separator was a setting on an instance of a converter.

```javascript
class Converter {
  constructor(separator) {
    this.separator = separator;
  }

  toArray(str) {
    return string.split(this.separator);
  }
}
```

In that case it would make a lot more sense to use class. You get me bro? :)

Reply:
I do like how a class looks though... But I get that in this case it may be better without ;(

Reply:
Both are equally valid my bro. A standalone function Is my preference in this specific instance. :P

I think I would feel better for a class with a static class if it was more concrete.

For example `Array.fromString`

Reply:
I get you👍

Reply:
You could also "monkey patch" the `Array` class itself. This is generally frowned upon in javascript, but quite common in other languages such as ruby. The reason it is frowned upon is that it **may** overwrite an existing method and you won't know about it. In your project that might be a concern though.

`Array.prototype` refers to the object which instances of string will look for methods on. You could also define static methods directly on `Array`.

```javascript
// A normal function is used here purposefully. Thanks to it "this" will be bound to the string instance.
String.prototype.toArray = function(separator = ',') { 
  return this.split('separator)
}

const str = "hi,there,buddy"
str.toArray()
//=> ["hi", "there", "buddy"]
```